openapi/postman/README.md

```
@@ -34,9 +34,13 @@ npx --yes newman run openapi/postman/assistant-api/assistant-api.smoke.postman_c
   --folder "Smoke Flow" \
   --bail \
   --env-var baseUrl=http://localhost:9007 \
-  --env-var authToken="$AUTH_TOKEN"
+  --env-var authToken="$AUTH_TOKEN" \
+  --env-var authId="$AUTH_ID" \
+  --env-var projectId="$PROJECT_ID"
 ```
 
+`authToken` must be the raw Rapida auth token without a `Bearer ` prefix. The assistant API also requires the authenticated user ID in `x-auth-id` and the selected project ID in `x-project-id`.
```

markdownlint-cli2 (0.23.2)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)